src/node/coverage.ts

```diff
@@ -0,0 +1,267 @@
+// Future goals:
+// - Persist owner/repo names in workspace storage
+// - Cache coverage
+// - Show coverage totals somewhere
+// - Maybe add codecov button to disable and/or view in codecov
+
+import {
+  ExtensionContext,
+  OverviewRulerLane,
+  Position,
+  Range,
+  Uri,
+  commands,
+  window,
+  workspace,
+} from "vscode";
+import axios from "axios";
+
+type Coverage =
+  | {
+      line_coverage: [number, number][];
+    }
+  | undefined;
+
+const Colors = {
+  covered: "rgb(33,181,119)",
```

Comment:
What would be the lift required to add custom color configurations in settings too :O 

Just a thought, maybe a super "dumb" env variable that requires the color to be in the same format as these

Reply:
Made issue for followup